perfmetrics/scripts/generate_files.py

```diff
@@ -26,7 +27,14 @@ def logmessage(message) -> None:
 def generate_files_and_upload_to_gcs_bucket(destination_blob_name, num_of_files,
                                             file_size_unit, file_size,
                                             filename_prefix,
-                                            local_destination_folder):
+                                            local_destination_folder,
+                                            upload_to_gcs_bucket,
+                                            logging_method, log):
```

Comment:
we shouldn't pass logging variables are arguments. every class should define its own logging variable.

Reply:
done